extension/src/lambda.js

```diff
@@ -0,0 +1,25 @@
+const utils = require('./utils')
+const paymentHandler = require('../src/paymentHandler/payment-handler')
+const setup = require('./config/init/ensure-resources')
+
+const logger = utils.getLogger()
+let init = false;
+
+exports.handler = async function(event) {
+  try {
+
+    init = init || await setup.ensureResources();
+    
+    const paymentResult = await paymentHandler.handlePayment(event.resource.obj)
+
+    return {
+      responseType: paymentResult.success ? `UpdateRequest` : `FailedValidation`,
```

Comment:
Hi,
Can you advise in which file we should document this? Or if we should just add a new one for Lambda responses (I don't see anything for the http endpoint so trying to find best place). Thanks

Reply:
@Max-John I guess it is not necessary as responseType is being seen and evaluated by Commercetools API only so the caller will rather see an error returned by API, which structure is described [here](https://docs.commercetools.com/http-api-projects-api-extensions#the-api-extension-within-the-flow-of-the-api-call). Maybe we could link as separate point under https://github.com/commercetools/commercetools-adyen-integration/blob/master/extension/docs/IntegrationGuide.md#error-cases:

`For more info on API-Extensions error handling please see [here](https://docs.commercetools.com/http-api-projects-api-extensions#the-api-extension-within-the-flow-of-the-api-call)`

Additionally if you have any input it could be useful to extend our best practices into 2 sections ("AWS Lambda" and "Hosted deployment"):
https://github.com/commercetools/commercetools-adyen-integration/blob/master/docs/BEST_PRACTICES.md

Reply:
@lojzatran Do you know why we did not provide in response `responseType` before? According to API doc it seem mandatory: https://docs.commercetools.com/http-api-projects-api-extensions#the-api-extension-within-the-flow-of-the-api-call

Reply:
@butenkor With HTTP extensions the responseType is taken care of using HTTP status codes 200, 201 and 400. Lambdas only need this as they have no concept of status code responses.

Reply:
I see. Thanks